addons/binding/org.openhab.binding.pioneeravr/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: PioneerAvr Binding
+Bundle-SymbolicName: org.openhab.binding.pioneeravr;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ gnu.io;resolution:=optional,
+ org.apache.commons.lang;version="2.6.0",
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.jupnp.model.types,
+ org.osgi.service.cm;version="1.5.0",
+ org.osgi.service.component;version="1.2.2",
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.pioneeravr,
+ org.openhab.binding.pioneeravr.discovery,
+ org.openhab.binding.pioneeravr.handler

```

Comment:
you should either also export org.openhab.binding.pioneeravr.protocol or move its classes into an internal package.